examples/tpm/tpm_key_helpers.h

```diff
@@ -0,0 +1,71 @@
+/* tpm_test_keys.h
```

Comment:
Please correct comment name. Please add `#ifdef WOLFSSH_TPM` around all this. Please remove and cleanup for functions that are not used.